packages/codegen-utils/vite.config.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from "vitest/config";
+import typescript from "@rollup/plugin-typescript";
+import peerDepsExternal from "rollup-plugin-peer-deps-external";
+
+/**
+ * @see https://vitejs.dev/config/
+ */
+export default defineConfig({
+  build: {
+    ssr: true,
+    lib: {
+      entry: "src/index.ts",
+      formats: ["cjs", "es"],
+    },
+    outDir: "dist",
+    minify: false,
+    sourcemap: true,
+  },
+  /**
+   * @see https://vitest.dev/config/
+   */
+  test: {
+    environment: "node",
+    globals: true,
+  },
+  plugins: [(typescript as any)(), peerDepsExternal()],
```

Comment:
why do we need to cast to `any` here?

Reply:
I just copied from @roninjin10's PR, the plugin has bad types for some reason